src/end.sol

```diff
@@ -283,11 +312,29 @@ contract End is LibNote {
         tag[ilk] = wdiv(spot.par(), uint(pip.read()));
     }
 
+    function halt(bytes32 ilk, uint256 id) external note {
```

Comment:
Okay let the bikeshedding begin: I prefer `sqip`, or maybe `scip`, here myself.

Reply:
I was thinking that, but my only issue is that since `sqip, scip and skip` are homonyms it might get confusing when we are discussing the functions verbally. If we want to maintain similarity with `skip` I'd personally prefer `swip`.

Reply:
Hmmmm okay fair, although I'd kind of think it would be either obvious from context or easy to clarify verbally.

In the homonym arena, I also quite like `skyp`. We could even pronounce it "skype" in verbal contexts to avoid ambiguity.

In the non-homonym arena, `snip` might be a little better mnemonically than `swip` (I certainly dislike `halt`, it gives the wrong impression IMO).

Reply:
I like `snip` actually, that's my fave so far. I'm also okay with `skyp`.